mix.exs

```diff
@@ -20,7 +21,7 @@ defmodule PLDS.MixProject do
   def application do
     [
       mod: {PLDS.Application, []},
-      extra_applications: [:logger, :runtime_tools]
+      extra_applications: [:logger, :runtime_tools, :os_mon, :inets]
```

Comment:
What do we need inets for? 🤔

Reply:
For httpc, to check if plds is already running or the port is in use.